sys/crypto/aes.c

```diff
@@ -49,8 +71,12 @@ static const cipher_interface_t aes_interface = {
     aes_encrypt,
     aes_decrypt
 };
+
+/* @deprecated */
```

Comment:
Please check if this is actually compiled by doxygen. I think it is `/**` for the start. Furthermore, please mention the replacement in the doxygen comment. Have a look at [`CONFIG_ASYMCUTE_BUFSIZE_EXP`](http://doc.riot-os.org/group__net__asymcute__conf.html#ga1947f6d1ba0c46781cb335a6c5bad1d6) as an example.

Reply:
A removal date is needed, place it after two release cycles, so after 2021.07

Reply:
Fixed